packages/agent-nodejs-testing/src/integrations/index.ts

```diff
@@ -14,15 +17,14 @@ export async function createTestableAgent<TypingsSchema extends TSchema = TSchem
   options?: TestableAgentOptions,
 ): Promise<TestableAgent<TypingsSchema>> {
   const port = options?.port || (await getAvailablePort());
-  // we don't want to pass the port to the agent because the port is not a valid agent option
-  delete options?.port;
```

Comment:
We can keep port inside options, it will improve the code by simplify some signatures